src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result;
+
+  if (days < 3) {
+    result = days * 40;
+  } else if (days >= 3 & days < 7) {
+    result = (days * 40) - 20;
+  } else if (days >= 7) {
+    result = (days * 40) - 50;
+  }
```

Comment:
what a magic numbers 🪄
lets create variables and then use them
it is better for managing them from one place and for understanding what do they mean

Reply:
avoid such nesting `else if` conditions
This code is hard to read and maintain